package.json

```diff
@@ -241,13 +240,7 @@
   "remarkConfig": {
     "plugins": [
       "preset-lint-recommended",
-      "preset-lint-consistent",
-      [
-        "lint-no-missing-blank-lines",
```

Comment:
Was this plugin causing issues? Prettier never adds empty lines. And we might want them after headings, for example.

Reply:
> Was this plugin causing issues? Prettier never adds empty lines.

The plugin was redundant as, unlike with JavaScript, Prettier adds empty lines in markdown.

For example:

```md
title
---
1) list
```js
var = x;
\```
Paragraph
```

Becomes:

```md
# title

1. list

```js
var = x;
\```

Paragraph
```

 It's lovely to work with.

Reply:
Nice!